Classes/Issues/Comments/Markdown/CommentModelsFromMarkdown.swift

```diff
@@ -404,20 +431,56 @@ func shortenGitHubLinks(attributedString: NSAttributedString,
     return mutableAttributedString
 }
 
+private let checkedIdentifier = ">/CHECKED>/"
+private let uncheckedIdentifier = ">/UNCHECKED>/"
+private let checkmarkRegex = try! NSRegularExpression(pattern: "^- (\\[([ |x])\\])", options: [.anchorsMatchLines])
```

Comment:
FWIW you can use checkmarks in numbered lists also, this regex will ignore those!

Reply:
Hmm, am I doing something wrong [here](https://github.com/rnystrom/GitHawk/issues/1074#issuecomment-346913669)? I tried testing numbered lists.

Reply:
I've updated the comment, and it looks like if you add a word after the checklist with a number then it works - but it's a bit buggy because if you look it splits them up so it's not one list.

This actually might be a GitHub bug!

Reply:
Ya, looks bizarre. I'm going to skip for now.